domains/guillelm.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Personal Portfolio Tech Website...",
+    "repo": "https://github.com/guilleLM71/portafolio",
+    "owner": {
+      "username": "guilleLM71",
+      "email": "jlauram3@fcpn.edu.bo"
+    },
+    "record": {
+      "CNAME": "https://guillelm71.github.io/portafolio"
```

```suggestion
      "CNAME": "guillelm71.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).